test/connectivity/scripts/connectivityTest.sh

```diff
@@ -228,6 +240,9 @@ function process_args() {
                 '-upstreamProtocol' ) saveNextArg=19;;
                 '-deploymentTestUpdatePeriod' ) saveNextArg=20;;
                 '-timeForReportingGeneration' ) saveNextArg=21;;
+                '-metricsEndpointsCSV' ) saveNextArg=22;;
+                '-metricsScrapeFrequencyInSecs' ) saveNextArg=23;;
+                '-metricsUploadTarget' ) saveNextArg=24;;
                 '-waitForTestComplete' ) WAIT_FOR_TEST_COMPLETE=1;;
 
                 '-cleanAll' ) CLEAN_ALL=1;;
```

Comment:
Please help to add "eventHubConnectionString" field in below required parameters checking.

Also default value for log type is set to "connectivity"; please change it to "testMetrics".

Reply:
The log type of "connectivity" is for test _reporting_, not metrics, so it should not be using the "testMetrics" log type. Because all the metrics are of the same type, the log type for "testMetrics" is hardcoded in the deployment file.

Reply:
yes, you are right.  Please help to add 1 required field checking for "eventHubConnectionString".  thanks.